src/referencePackages/src/microsoft.dotnet.templatelocator/5.0.100-rc.1.20421.19/microsoft.dotnet.templateLocator.csproj

```diff
@@ -0,0 +1,68 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net472</TargetFrameworks>
+    <NuspecFile>$(ArtifactsBinDir)microsoft.dotnet.templatelocator/5.0.100-rc.1.20421.19/microsoft.dotnet.templatelocator.nuspec</NuspecFile>
+    <AssemblyOriginatorKeyFile>$(KeyFileDir)35MSSharedLib1024.snk</AssemblyOriginatorKeyFile>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <OutputPath>$(ArtifactsBinDir)microsoft.dotnet.templatelocator/5.0.100-rc.1.20421.19/ref/</OutputPath>
+    <IntermediateOutputPath>$(ArtifactsObjDir)microsoft.dotnet.templatelocator/5.0.100-rc.1.20421.19</IntermediateOutputPath>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+    <OutputPath>$(ArtifactsBinDir)microsoft.dotnet.templatelocator/5.0.100-rc.1.20421.19/lib/</OutputPath>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+    <OutputPath>$(ArtifactsBinDir)microsoft.dotnet.templatelocator/5.0.100-rc.1.20421.19/lib/</OutputPath>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+    <OutputPath>$(ArtifactsBinDir)microsoft.dotnet.templatelocator/5.0.100-rc.1.20421.19/lib/</OutputPath>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'net472' ">
+    <OutputPath>$(ArtifactsBinDir)microsoft.dotnet.templatelocator/5.0.100-rc.1.20421.19/lib/</OutputPath>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'net472' ">
+    <OutputPath>$(ArtifactsBinDir)microsoft.dotnet.templatelocator/5.0.100-rc.1.20421.19/lib/</OutputPath>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'net472' ">
+    <OutputPath>$(ArtifactsBinDir)microsoft.dotnet.templatelocator/5.0.100-rc.1.20421.19/lib/</OutputPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="**/ref/$(TargetFramework)/*.cs" />
+    <Compile Include="**/lib/$(TargetFramework)/*.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+    <PackageReference Include="NETStandard.Library" Version="$(NETStandardLibraryPackageVersion)" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+    <PackageReference Include="NETStandard.Library" Version="$(NETStandardLibraryPackageVersion)" />
+  </ItemGroup>
```

Comment:
It doesn't look like this is one of the things causing CI to fail, but huh? Is this maybe because there are multiple assemblies in this package--is the tooling meant to handle that? @dseefeld

Reply:
The tooling should handle that fine.  I'm not sure what is causing that.

Reply:
I think the extras should be removed.